### PR TITLE
Dev portal revamp

### PR DESCRIPTION
**Pull request by rav3**

### CODE REVIEW

File: _layouts/layout-page-sidenav-sharable.html [hunk 1]
```diff
@@ -3,23 +3,22 @@
...
-  <div class="col is-3 is-hidden-touch sidenav">
+  <div class="col is-2 is-hidden-touch sidenav">
     <aside class="sgds-menu">
       <ul class="sgds-menu-list">
         {%- include sidenav.html -%}
       </ul>
     </aside>
   </div>
-  <div class="col is-8 is-12-touch">
+  <div class="col is-8 is-offset-1 is-12-touch">
```
Comment (teochengyong) [inline _layouts/layout-page-sidenav-sharable.html:13]:
```suggestion
  <div class="col is-8 is-offset-1 is-12-mobile">
```
Suggests to use `is-12-touch` as when the view point < 1023px is-12-touch is making the width 100% and with the margin-left from offset-1 it is pushing the columns out of the viewport. 

`is-12-touch`
image

`is-12-mobile`
image

File: _layouts/layout-page-sidenav.html [hunk 2]
```diff
@@ -3,39 +3,39 @@
...
+  <div class="col is-2 is-hidden-touch sidenav">
     <aside class="sgds-menu">
       <ul class="sgds-menu-list">
         {%- include sidenav.html -%}
       </ul>
     </aside>
   </div>
   {% if page.sharable == true or page.sharable == nil %}
-  <div class="col is-8 is-12-touch">
+  <div class="col is-8 is-offset-1 is-12-touch">
```
Comment (teochengyong) [inline _layouts/layout-page-sidenav.html:14]:
```suggestion
  <div class="col is-8 is-offset-1 is-12-mobile">
```

File: _layouts/layout-page-sidenav.html [hunk 3]
```diff
@@ -3,39 +3,39 @@
...
       </em>
     </p>
+    <div class="small-viewport-social-media-buttons">
+      {% include social-media.html display="mobile" %}
+    </div>
+    {%- include feedback.html -%}
   </div>
   {% else %}
-  <div class="col is-9 is-12-touch">
+  <div class="col is-9 is-offset-1 is-12-touch">
```
Comment (teochengyong) [inline _layouts/layout-page-sidenav.html:27]:
```suggestion
  <div class="col is-9 is-offset-1 is-12-mobile">
```

File: apps/src/search/SearchApp.vue [hunk 4]
```diff
@@ -14,20 +14,18 @@
     </div>
 
     <div class="sgds-container search-content" v-if="!searching">
-      <h2>
+      <h2 class="margin--bottom">
```
[the remark this answers is not in the document]
Comment (rav3) [reply apps/src/search/SearchApp.vue:17]: Yes. This shows the search query thats entered by the user.

File: search/search.html [hunk 5]
```diff
@@ -4,13 +4,14 @@
...
+<section class="sgds-section breadcrumb-title-section has-background-general">
+    <div class="wave-gfx"></div>
     <div class="sgds-container">
-        <h2 class="has-text-white ">{{ page.title }}</h2>
+        <h1 class="has-text-white"><b>{{ page.title }}</b></h1>
     </div>
     <div class="sgds-container">
-        <form action="/search/" method="get">
-            <div class="row padding--bottom--lg padding--top--lg">
+        <!-- <form action="/search/" method="get">
```
[the remark this answers is not in the document]
Comment (rav3) [reply search/search.html:13]: Form will show only for tablet and below.

**Review by teochengyong — APPROVED** [review]
All looks good 😄

**Pull request merged**